examples/mcu-embassy/Cargo.toml

```diff
@@ -0,0 +1,118 @@
+# Copyright © 2025 David Haig
+# SPDX-License-Identifier: MIT
+
+[package]
+name = "mcu-embassy"
+version = "0.1.0"
+edition = "2021"
+authors = ["David Haig <david@ninjametal.com>"]
+readme = "README.md"
+resolver = "2"
+license = "MIT"
+publish = false
+repository = "https://github.com/slint-ui/slint"
+homepage = "https://slint.dev"
+
+[dependencies]
+log = "0.4"
+defmt = { version = "0.3", optional = true, features = ["alloc"] }
+defmt-rtt = { version = "0.4", optional = true }
+panic-probe = { version = "0.3", optional = true, features = ["print-defmt"] }
+cortex-m = { version = "0.7.7", optional = true, features = [
+    "inline-asm",
+    "critical-section-single-core",
+] }
+cortex-m-rt = { version = "0.7.3", optional = true }
+slint-generated = { path = "./slint_generated" }
+embedded-alloc = { version = "0.5", optional = true }
+heapless = { version = "0.8", default-features = false, features = [
+    "defmt-03",
+] }
+static_cell = { version = "2", optional = true }
+
+slint = { version = "=1.9.0", path = "../../api/rs/slint", default-features = false, features = [
+    "compat-1-2",
+    "unsafe-single-threaded",
+    "libm",
+    "renderer-software",
+] }
+i-slint-core-macros = { version = "=1.9.0", path = "../../internal/core-macros" }
```

Comment:
This should not be needed :)

```suggestion
```

Reply:
Thanks @tronical, I'm just in the process of fixing a small Embassy time-driver dependency issue. Once I have figured it out I will include the suggestions above too.

Reply:
Done